package.json

```diff
@@ -111,7 +111,12 @@
     "@types/babel__traverse": "ts3.5",
     "@types/node-fetch": "ts3.5",
     "@types/yargs": "^15.0.5",
-    "glob": "^9.3.5"
+    "test-exclude": {
```

Comment:
Only override glob for specific transitive deps

Reply:
Do we plan to bump 'test-exclude' and 'tslint' later?